src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/db/Orderly.kt

```diff
@@ -28,50 +22,43 @@ class Orderly(val isReviewer: Boolean,
             // create a temp table containing the latest version ID for each report name
             val latestVersionForEachReport = getLatestVersionsForReports(it)
 
-            val versions =
-                    it.dsl.withTemporaryTable(latestVersionForEachReport)
-                            .select(REPORT_VERSION.REPORT,
-                                    REPORT_VERSION.DISPLAYNAME,
-                                    REPORT_VERSION.ID,
-                                    REPORT_VERSION.PUBLISHED,
-                                    REPORT_VERSION.DATE,
-                                    latestVersionForEachReport.field<String>("latestVersion")
-                            )
-                            .from(REPORT_VERSION)
-                            .join(latestVersionForEachReport.tableName)
-                            .on(REPORT_VERSION.REPORT.eq(latestVersionForEachReport.field("report")))
-                            .where(shouldIncludeReportVersion)
-                            .orderBy(REPORT_VERSION.REPORT, REPORT_VERSION.ID)
-                            .fetch()
+            val versions = it.dsl.withTemporaryTable(latestVersionForEachReport)
+                    .select(REPORT_VERSION.REPORT.`as`("name"),
+                            REPORT_VERSION.DISPLAYNAME,
+                            REPORT_VERSION.ID,
+                            REPORT_VERSION.PUBLISHED,
+                            REPORT_VERSION.DATE,
+                            latestVersionForEachReport.field<String>("latestVersion"),
+                            REPORT_VERSION.DESCRIPTION
+                    )
+                    .from(REPORT_VERSION)
+                    .join(latestVersionForEachReport.tableName)
+                    .on(REPORT_VERSION.REPORT.eq(latestVersionForEachReport.field("report")))
+                    .where(shouldIncludeReportVersion)
+                    .orderBy(REPORT_VERSION.REPORT, REPORT_VERSION.ID)
+                    .fetchInto(BasicReportVersion::class.java)
 
             return mapToReportVersions(it, versions)
         }
     }
 
-    override fun getGlobalPinnedReports(): List<ReportVersion>
+    override fun getGlobalPinnedReports(): List<Report>
     {
         JooqContext().use {
 
-            // create a temp table containing the latest visible version ID for each report name
-            val latestVersionForEachReport = getLatestVersionsForReports(it)
-
-            val versions =  it.dsl.withTemporaryTable(latestVersionForEachReport)
-                    .select(REPORT_VERSION.REPORT,
+            val versions = it.dsl
```

Comment:
Also noticed that for the pinned reports we were fetching these whole complex `ReportVersion` objects with aretfacts, custom fields etc when the only properties needed are the date, id and display name. So simplified this method and changed to return `Report` objects.